sdk/storage/azure-storage-blob-stress/src/main/java/com/azure/storage/blob/stress/UploadFromFile.java

```diff
@@ -72,7 +75,9 @@ protected Mono<Void> runInternalAsync(Context span) {
                 return Mono.using(
                     () -> downloadPath.resolve(UUID.randomUUID() + ".txt"),
                     path -> asyncClient.uploadFromFileWithResponse(new BlobUploadFromFileOptions(uploadFilePath.toString())
-                            .setParallelTransferOptions(new ParallelTransferOptions().setMaxSingleUploadSizeLong(4 * 1024 * 1024L)))
+                            .setParallelTransferOptions(new ParallelTransferOptions()
+                                .setMaxConcurrency(parallelTransferOptions.getMaxConcurrency())
+                                .setMaxSingleUploadSizeLong(4 * 1024 * 1024L)))
```

Comment:
Same suggestion here. I'm wondering if we can reuse the parallelTransferOptions defined in the constructor instead of creating a new one?